src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventAttachments/groupEventAttachmentsTableRow.tsx

```diff
@@ -19,64 +19,63 @@ type Props = {
   isDeleted: boolean;
 };
 
-class GroupEventAttachmentsTableRow extends React.Component<Props> {
-  getEventUrl() {
-    const {attachment, orgId, groupId} = this.props;
-
+const GroupEventAttachmentsTableRow = ({
+  attachment,
+  projectId,
+  onDelete,
+  isDeleted,
+  orgId,
+  groupId,
+}: Props) => {
+  const getEventUrl = () => {
```

Comment:
Does this  esd to be a function? It looks like it is invariant.

Reply:
Not really, I've changed it, thanks 👍